### PR TITLE
More hails

### PR DESCRIPTION
**Pull request by ph2000bis**
Added some hails for friendly hai unfettered and specific hails for bounty.

### CODE REVIEW

File: data/hai.txt [hunk 1]
```diff
@@ -460,6 +460,48 @@ phrase "hostile hai"
...
+		"accept "
+		"tolerate "
+	word
+		"true"
+		"strong"
+		"ambitious"
+	word
+		" races"
+		" people"
+		" individuals"
```
Comment (Bladewood) [inline data/hai.txt]: It's preferred to put another "word" line in between that just contains a single space. [Observe.](https://github.com/endless-sky/endless-sky/blob/master/data/hails.txt#L1420)

File: data/hai.txt [hunk 2]
```diff
@@ -460,6 +460,48 @@ phrase "hostile hai"
...
+		"ready to"
+		"willing to"
+		"prepared to"
+	word
+		" help"
+		" assist"
+		" work with"
+		" support"
+		" follow"
+		" obey to"
```
Comment (Amazinite) [inline data/hai.txt]: obey to? Would just "obey" not be better?

File: data/hai.txt [hunk 3]
```diff
@@ -465,19 +465,27 @@ phrase "friendly unfettered"
 	word
 		"We "
```
Comment (Bladewood) [inline data/hai.txt]: Still have to deal with this. https://github.com/endless-sky/endless-sky/blob/master/data/hails.txt#L1387

File: data/hails.txt [hunk 4]
```diff
@@ -2431,19 +2435,29 @@ phrase "hostile bounty"
 		"imagined"
 		"dreamed"
 	word
-		" that "
+		" "
+	word
+		"that"
+		""
```
Comment (Amazinite) [inline data/hails.txt]: This "" between two " "'s can cause a double space.

File: data/hails.txt [hunk 5]
```diff
@@ -2390,9 +2390,13 @@ phrase "hostile bounty"
 		"weakling"
 		"worthless wretch"
 	word
-		" they "
-		" the government "
-		" the law "
+		" "
+	word
+		"they "
```
Comment (Amazinite) [inline data/hails.txt]: Extra space after they causes a double space

File: data/hai.txt [hunk 6]
```diff
@@ -465,19 +465,27 @@ phrase "friendly unfettered"
...
-		" individuals"
+		" "
+	word
+		"races"
+		"people"
+		"individuals"
+	word
+		" "
+	word
+		"that are"
```
Comment (Amazinite) [inline data/hai.txt]: Another double space scenario

File: data/hai.txt [hunk 7]
```diff
@@ -465,19 +465,27 @@ phrase "friendly unfettered"
...
+		""
```
Comment (Amazinite) [inline data/hai.txt]: Another double space scenario

File: data/hails.txt [hunk 8]
```diff
@@ -2363,6 +2363,128 @@ phrase "hostile pirate"
...
+		"believed"
+		"hoped"
+		"expected"
+		"imagined"
+		"dreamed"
+	word
+		" "
+	word
+		"that"
+		""
```
Comment (jafdy) [inline data/hails.txt]: Double space situation here.

File: data/hai.txt (hunk 3 above)
Comment (ph2000bis) [inline data/hai.txt]: https://github.com/endless-sky/endless-sky/blob/master/data/hails.txt#L1593

File: data/hails.txt [hunk 9]
```diff
@@ -2435,11 +2435,9 @@ phrase "hostile bounty"
 		"imagined"
 		"dreamed"
 	word
+		" that "
 		" "
 	word
-		"that"
-		""
-	word
 		" "
```
Comment (Amazinite) [inline data/hails.txt]: This one can still double space.
"If you dreamed  I..."

File: data/hai.txt (hunk 3 above)
Comment (Bladewood) [inline data/hai.txt]: ...goddammit @Amazinite

**Comment by ph2000bis** [comment]
Go home @ph2000bis you're drunk. Oh wait, that's me.

**Comment by KiLEdEnNis** [comment]
no drinking on Github

**Pull request merged**